forc-pkg/Cargo.toml

```diff
@@ -11,6 +11,7 @@ description = "Building, locking, fetching and updating Sway projects as Forc pa
 [dependencies]
 anyhow = "1"
 forc-util = { version = "0.16.2", path = "../forc-util" }
+fuel-tx = "0.12"
```

Comment:
`StorageSlot` should be under `fuel-types` (not `fuels-types`) I think. Can you file an issue to follow-up on moving that? We probably don't want to introduce a dependency on `fuel-tx` just for a type.

Reply:
Done 
https://github.com/FuelLabs/sway/issues/2115
https://github.com/FuelLabs/fuel-tx/issues/152